### PR TITLE
set up packet page

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
- added update to rspec functional testing to go through proper new routes, karma tests for each card on the packet page. 

#### Which User story does this relate to?
- https://www.pivotaltracker.com/story/show/152095524

#### Brief feature description
- this PR will add the default packet overview page that the user will click into when creating a new rfa application.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- rspec: 85%,
- karma: 67%

#### JS errors diff as a result of this PR?
- 8 -> 8

### CODE REVIEW

File: spec/controllers/welcome_controller_spec.rb [hunk 1]
```diff
@@ -3,11 +3,10 @@
...
 describe WelcomeController do
   before(:each) do
     allow(controller).to receive_messages(:authenticate_with_cwds => true)
-    allow(controller).to receive_messages(:get_session_token => static_token)
+    allow(controller).to receive_messages(:get_session_token => ENV['TOKEN'])
```
Comment (ratneshraval) [inline spec/controllers/welcome_controller_spec.rb]: Nice

File: .codeclimate.yml [hunk 2]
```diff
@@ -4,47 +4,47 @@ engines:
...
   duplication:
-    enabled: true
     config:
       languages:
-      - ruby
-      - javascript
-      - python
-      - php
-  eslint:
+        javascript:
```
[the remark this answers is not in the document]
Comment (lauherkcambria) [reply .codeclimate.yml]: for sure

**Comment by lauherkcambria** [comment]
@ratneshraval good catch with those warnings, i've removed them.

File: spec/controllers/welcome_controller_spec.rb (hunk 1 above)
Comment (lauherkcambria) [reply spec/controllers/welcome_controller_spec.rb]: thankssssss

File: app/javascript/rfa_forms/rfa_packet_list/lic198bOverview.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        focusClassName={this.props.getFocusClassName('Lic198BOverview') + ' ' + 'card phone-section double-gap-top'}>
+        <span>default Lic198B </span>
+      </CardLayout>
+
+    )
+  }
+}
+
+Lic198BOverview.propTypes = {
+  applicationId: PropTypes.string,
```
Comment (anuroop15) [inline app/javascript/rfa_forms/rfa_packet_list/lic198bOverview.js]: are you using this applicationId in the component?

File: test/javascript/rfa_forms/rfa_packet_list/lic198bOverview_tests.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <Lic198BOverview
+      focusComponentName={'Rfa01BOverview'}
+      applicationId={'10'}
+      setFocusState={setFocusStateSpy}
+      getFocusClassName={getFocusClassNameSpy}
+   />)
+  it('To Load with the correct label', () => {
+    expect(rfaPacketRenderedView.props.label).toBe('Lic198B Section Summary')
+  })
+})
```
Comment (anuroop15) [inline test/javascript/rfa_forms/rfa_packet_list/lic198bOverview_tests.js]: good to have a test case for focused component on click method

File: app/javascript/rfa_forms/rfa_packet_list/lic198bOverview.js (hunk 3 above)
Comment (lauherkcambria) [reply app/javascript/rfa_forms/rfa_packet_list/lic198bOverview.js]: we will need it when this part of the form is complete.

File: test/javascript/rfa_forms/rfa_packet_list/lic198bOverview_tests.js (hunk 4 above)
Comment (lauherkcambria) [reply test/javascript/rfa_forms/rfa_packet_list/lic198bOverview_tests.js]: will make a chore for this, but this shouldn't prevent the pr from being merged.

**Pull request merged**